app/src/main/java/com/uc/mymovies/dto/Movie.kt

```diff
@@ -1,5 +1,15 @@
 package com.uc.mymovies.dto
 
+/**
+ * Data class representing a movie object.
+ *
+ * @param title
+ * @param genre
+ * @param director
+ * @param image
+ * @param synopsis
+ * @param isFavorite
+ */
```

Comment:
Simple KDocs

Reply:
If adding KDoc, complete it fully.  Give a description to the parameters, for instance.
KDoc that reiterates what is readable from code does not add value.